app/components/Home/Section/Values.tsx

```diff
@@ -41,10 +41,10 @@ function TopEffect() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [])
     return (
-        <div ref={refTop} className="text-center w-full rounded-t-xl bg-green">
-            <motion.div style={{ marginLeft }} className="overflow-hidden whitespace-nowrap font-trash text-lg font-bold uppercase md:text-4xl xl:text-6xl pt-3">We align the best interests</motion.div>
-            <motion.div style={{ marginRight }} className="overflow-hidden whitespace-nowrap font-americana text-xl uppercase md:text-3xl xl:text-5xl">of investors, business</motion.div>
-            <motion.div style={{ marginLeft }} className="overflow-hidden whitespace-nowrap font-trash text-lg font-bold uppercase md:text-4xl xl:text-6xl lg:pt-3">and the planet</motion.div>
+        <div ref={refTop} className="text-center w-screen  bg-green ">
+            <motion.div style={{ marginLeft }} className="overflow-hidden whitespace-nowrap font-trash text-lg font-bold uppercase md:text-4xl xl:text-7xl 2xl:text-8xl pt-3 w-full"><p>We align best interests</p></motion.div>
```

Comment:
w-full is not needed.
Ok to have a bigger size for xl and beyond.
What about default (mobile), md and large format?

Reply:
Issue here is if I dont put w-full, text is hidden at this stage :
<img width="1406" alt="CleanShot 2022-11-17 at 10 59 54@2x" src="https://user-images.githubusercontent.com/25151724/202416395-e97d7595-384e-45b1-85fb-660ded35eaf7.png">

If a add w-full for line 45 & 47 it works better. : 
<img width="1469" alt="CleanShot 2022-11-17 at 11 01 45@2x" src="https://user-images.githubusercontent.com/25151724/202416692-8ee2b7c9-3138-4029-890c-f0916a90048d.png">

For line 46 middle line w-full seems to break the animation > can't figure out why

Reply:
Ok @schwepps  now line 46 is working as I added a grid properties with justify items center
<img width="756" alt="CleanShot 2022-11-17 at 11 08 26@2x" src="https://user-images.githubusercontent.com/25151724/202418130-11e1588c-24cc-49bf-86b7-42371140ab79.png">